tensorboard/webapp/core/effects/core_effects.ts

```diff
@@ -70,28 +86,55 @@ export class CoreEffects {
     return from(this.tfBackend.ref.runsStore.refresh());
   }
 
+  private readonly onDashboardLoad$ = merge(
+    this.actions$.pipe(ofType(coreLoaded)),
+    this.actions$.pipe(
+      ofType(navigated),
+      filter(({after}) => {
+        return (
+          after.routeKind === RouteKind.COMPARE_EXPERIMENT ||
+          after.routeKind === RouteKind.EXPERIMENT
+        );
+      }),
+      withLatestFrom(this.store.select(getRouteId)),
+      distinctUntilChanged(([, beforeRouteId], [, afterRouteId]) => {
+        return beforeRouteId === afterRouteId;
+      })
+    )
+  ).pipe(
+    throttleTime(DATA_LOAD_COND_THROTTLE_IN_MS, undefined, {leading: true})
+  );
+
+  // Emits when data should be refreshed.
+  // HACK: currently, plugins list loaded state is used as a proxy to know
```

Comment:
I think this comment could be a bit more explicit. Why is state === LOADING grounds for not updating the data?

Reply:
Since I am not really changing the existing behavior and the behavior is codified in the test, I will be terse with the description.